activity.py

```diff
@@ -207,6 +213,29 @@ def do(cls, activities):
     def cancel(cls, activities):
         pass
 
+    @classmethod
+    @ModelView.button
+    def createsubactivities(cls, activities):
+        pool = Pool()
+        Warning = pool.get('res.user.warning')
+        for activity in activities:
+            if('---' in activity.description):
```

Comment:
The if is not necessary because it does not guarantee we're going to split anyway.